### ISSUE TITLE
Trailing slash removed from record_session url

### ISSUE THREAD

**Issue by jrowen** [rstudio/shinyloadtest#112]
It appears that the trailing url slash is being removed from the `record_session` call.  In particular, I'm finding it missing from the url in [this line from `isProtected` (`build` function)](https://github.com/rstudio/shinyloadtest/blob/7caddac200f671c05ab55bf0b26d9d6c56d8457f/R/auth.R#L22), which in my case generates a 404 error and mistakenly prompts me to login.
```
shinyloadtest::record_session('https://shinyapp.example.com/?param=value')
```

**Comment by alandipert** [comment]
Thank you for the report. We'll look into it.

**Comment by alandipert** [comment]
Hi, looking at this now. When I test it out, `URLBuilder$build()` seems to do the right thing with the example URL you provided:

```
> url <- "https://shinyapp.example.com/?param=value"
> builder <- shinyloadtest:::URLBuilder$new(url)
> builder$build() == url
[1] TRUE
```
It looks also like `curl::curl_fetch_memory()` preserves the slash as well.

Would you please elaborate more on what you're seeing and the conditions under which you see it? I wonder if some other problem unrelated to URL construction is the culprit.

**Comment by alandipert** [comment]
If I understand this issue correctly, it was likely fixed by #119, so closing. Please don't hesitate to re-open if I'm wrong.
